editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -171,11 +171,48 @@ const ValueContainer = betterReactMemo(
   },
 )
 
-const filterOption = createFilter({ ignoreAccents: false })
+const filterOption = () => true
+const MaxResults = 500
 
 export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNameSelect', () => {
   const theme = useColorTheme()
   const dispatch = useEditorState((store) => store.dispatch, 'ClassNameSelect dispatch')
+  const [input, setInput] = React.useState('')
+  const filteredOptions = React.useMemo(() => {
```

Comment:
Does this imply that continued typing would lose / reset temporary selection (eg type three letters, use arrow keys to navigate, continue typing)? I'm guessing it would - but if so, does it 'feel wrong'?

Reply:
I've just tested and it doesn't. I tested by typing `text-`, using the arrow keys to highlight `text-blue-300`, then typed the letter 'b', and though the filtered results had updated (and the highlighted result wouldn't have been first), the highlighted result was still retained